M2/Macaulay2/m2/startup.m2.in

```diff
@@ -1,642 +0,0 @@
--- -*- coding: utf-8 -*-
---		Copyright 1993-2009 by Daniel R. Grayson
--- layout.m2, includes the former contents of ../m2/startup.m2
--- this file gets incorporated into the executable file bin/M2 as the string 'startupString'
--- we want to ignore the --datarootdir setting, whatever that means, so here we we mention it: @datarootdir@
-
-interpreterDepth = loadDepth = errorDepth = 0
-debuggingMode = true
-stopIfError = false
-notify = false
-debugWarningHashcode = null
-gotarg := arg -> any(commandLine, s -> s === arg)
-randomSeed = -- initializing this here so the startup is reproducible
-if gotarg "--no-randomize" then 0 else (currentTime() << 16) + processID()
-if gotarg "--notify" then notify = true
-if gotarg "--stop" then stopIfError = true
-if gotarg "--int"  then handleInterrupts = false
-
-firstTime := class PackageDictionary === Symbol
```

Comment:
I vaguely recall that at some point a running M2 process could be halted and "pickled" into a core dump in such a way that it could be restarted from that position, and the point of `firstTime` was to indicate whether we're loading for the first time or from a pickle. Is this correct?

Should we remove it everywhere?